docs/drain_account.md

```diff
@@ -16,13 +16,14 @@ Once we know the associated fees, we can calculate the maximum amount that needs
 Finally, we can do the transfer operation and use the maximum amount we just calculated as the `amount` parameter of the `transfer` function.
 
 :::note
-In the following example, we have not revealed the account that we want to empty. We need to keep in mind that there are fees related to a reveal operation. We are subtracting 1420 mutez from the balance to cover reveal fees.
+In the following example, we have not revealed the account that we want to empty. We need to keep in mind that there are fees related to a reveal operation. We are estimating the reveal fee and subtracting them from the balance to cover reveal fees.
```

Comment:
fees  instead of fee    or if fee then  it instead of them.

Reply:
fixed, thanks!